data/json/items/archery.json

```diff
@@ -184,32 +184,76 @@
       "damage" : 14,
       "pierce" : 7,
       "range" : 10,
-      "dispersion" : 60,
+	  "dispersion": 80,
       "loudness": 0,
       "count" : 10,
-      "effects" : ["RECOVER_15"]
+	  "effects": ["RECOVER_50"]
    },
+   {
+   	  "type": "AMMO",
+	  "id": "arrow_metal_bodkin",
+	  "price": 4000,
+	  "name": "metal bodkin arrow",
+	  "symbol": "=",
+	  "color": "green",
+	  "description": "This metal arrow has fletching and a steel bodkin arrowhead for armor penetration.  Stands a very good chance of remaining intact once fired.",
+	  "material": "steel",
+	  "volume": 1,
+	  "weight": 60,
+	  "bashing": 3,
+	  "cutting": 2,
+	  "ammo_type": "arrow",
+	  "damage": 10,
+	  "pierce": 15,
+	  "range": 12,
+	  "dispersion": 75,
+	  "loudness": 0,
+	  "count": 10,
+	  "effects": ["RECOVER_50"]
+   },
+   {
+	  "type": "AMMO",
+	  "id": "arrow_metal_target",
+	  "price": 4000,
+	  "name": "metal target arrow",
+	  "symbol": "=",
+	  "color": "green",
+	  "description": "This metal arrow has extra long fletching and a round, balanced arrowhead for extra accuracy, at the cost of some damage.  Stands a very good chance of remaining intact once fired.",
+      "material": "steel",
+	  "volume": 1,
+	  "weight": 50,
+	  "bashing": 3,
+	  "cutting": 2,
+	  "ammo_type": "arrow",
+	  "damage": 12,
+	  "pierce": 5,
+	  "range": 8,
+	  "dispersion": 50,
+	  "loudness": 0,
+	  "count": 10,
+	  "effects": ["RECOVER_60"]
+   }, 
    {
       "type" : "AMMO",
       "id" : "arrow_cf",
       "price" : 9500,
       "name" : "carbon fiber arrow",
       "symbol" : "=",
       "color" : "green",
-      "description" : "High-tech carbon fiber shafts and 100 grain broadheads.  Very lightweight, fast, and notoriously fragile.  Stands a bad chance of remaining intact once fired.",
+	  "description": "High-tech carbon fiber shafts and 100 grain broadheads. Very lightweight, fast, and durable. Stands a great chance of remaining intact once fired.",
       "material" : "kevlar",
       "volume" : 2,
-      "weight" : 8,
+      "weight" : 10,
       "bashing" : 1,
       "ammo_type" : "arrow",
-      "damage" : 10,
-      "pierce" : 5,
-      "range" : 8,
-      "dispersion" : 135,
+      "damage" : 16,
+      "pierce" : 8,
+      "range" : 14,
+      "dispersion" : 65,
       "loudness": 0,
       "count" : 8,
       "stack_size" : 10,
-      "effects" : ["RECOVER_3"]
+      "effects" : ["RECOVER_80"]
```

Comment:
What type of carbon fiber is this based off of? [This source](https://onlinelibrary.wiley.com/doi/full/10.1111/1556-4029.13503) indicates most carbon fiber arrows are quite fragile.
>Carbon fiber arrows have smaller diameters than their aluminum arrow counterparts due to the strength of this material. Their light weight allows for higher velocities and for longer travel distances. The disadvantage of these arrows is that the carbon fiber can be fragile and is not subject to reuse after damage

Reply:
In my experience CF arrows are far more durable. The main difference is that metal arrows will deform and can sometimes be fixed while CF will just shatter or sometime chip. It takes significantly more effort to damage a CF arrow over a metal one though.

Not sure what to say on diameter. In my experience most target shooters use thin metal or CF arrows, while hunters use almost exclusively thicker CF.